onebusaway-android/src/google/java/org/onebusaway/android/map/googlemapsv2/BaseMapFragment.java

```diff
@@ -727,6 +729,20 @@ public void deactivate() {
         mListener = null;
     }
 
+    /**
+     * Adds a marker to the Map
+     *
+     * @param l is the marker's position in the map
+     * @return the added marker object,
+     * the marker object can be used to remove the marker from the map by calling marker.remove()
+     */
+    public Marker addMarker(Location l) {
```

Comment:
@cagryInside This will still cause the `Marker` class to be imported into classes outside the `map` package.  I'll submit a PR shortly with a design that I believe will solve this problem and still allow the addition of markers from classes outside the package.